==== Unchecked return value from external call. ====
SWC ID: 104
Severity: Medium
Contract: LuckyDoubler
Function name: fallback
PC address: 1276
Estimated Gas Usage: 1105 - 35386
The return value of a message call is not checked.
External calls return a boolean value. If the callee halts with an exception, 'false' is returned and execution continues in the caller. The caller should check whether an exception happened and react accordingly to avoid unexpected behavior. For example it is often desirable to wrap external calls in require() so the transaction is reverted if the call fails.
--------------------
In file: ../results/smartbugs/sGuard/bad_randomness/lucky_doubler/lucky_doubler.sol:96

msg.sender.send(msg.value)

--------------------
Initial State:

Account: [CREATOR], balance: 0x0, nonce:0, storage:{}
Account: [ATTACKER], balance: 0x0, nonce:0, storage:{}

Transaction Sequence:

Caller: [CREATOR], calldata: , decoded_data: , value: 0x0
Caller: [CREATOR], function: changeMultiplier(uint256), txdata: 0xced926700000000000000000000000000000000000000000000000000000000000000078, decoded_data: (120,), value: 0x0
Caller: [SOMEGUY], function: blockHashAddendsInexpansible(uint256), txdata: 0x00000000, value: 0x0


